test.sh

```diff
@@ -119,8 +123,6 @@ r-unit() {
   # This one needs to be run from the root dir
   tests/compare_dist_test.R
 
-  tests/gen_counts_test.R
-
```

Comment:
Oops. It removed that to test the target. That test had been failing. I have reverted it.